### PR TITLE
Update configure-azure-cni.md

### PR DESCRIPTION
**Pull request by reknai**
PR for https://supportability.visualstudio.com/AzureContainers/_workitems/edit/80451

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@reknai - Can you resolve the warnings listed above?

@asudbring - Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/aks/configure-azure-cni.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This article shows you how to use Azure CNI networking to create and use a virtu
 
   The steps in this quickstart run the Azure PowerShell cmdlets interactively in [Azure Cloud Shell](/azure/cloud-shell/overview). To run the commands in the Cloud Shell, select **Open Cloudshell** at the upper-right corner of a code block. Select **Copy** to copy the code and then paste it into Cloud Shell to run it. You can also run the Cloud Shell from within the Azure portal.
 
-  You can also [install Azure PowerShell locally](/powershell/azure/install-azure-powershell) to run the cmdlets. The steps in this article require Azure PowerShell module version 5.4.1 or later. Run `Get-Module -ListAvailable Az` to find your installed version. If you need to upgrade, see [Update the Azure PowerShell module](/powershell/azure/install-Az-ps#update-the-azure-powershell-module).
+  You can also [install Azure PowerShell locally](/powershell/azure/install-azure-powershell) to run the cmdlets. The steps in this article require Azure PowerShell module version 5.4.1 or later. Run `Get-Module -ListAvailable Az` to find your installed version. If you need to upgrade for specific OS, see [macOS][macOSUpgrade], [Windows][windowsUpgrade] or [Linux][linuxUpgrade].
```
Comment (schaffererin) [inline articles/aks/configure-azure-cni.md:32]:
```suggestion
  You can also [install Azure PowerShell locally](/powershell/azure/install-azure-powershell) to run the cmdlets. The steps in this article require Azure PowerShell module version 5.4.1 or later. Run `Get-Module -ListAvailable Az` to find your installed version. If you need to upgrade for a specific OS, see [macOS][macOSUpgrade], [Windows][windowsUpgrade], or [Linux][linuxUpgrade].
```

**Review by schaffererin — CHANGES_REQUESTED** [review]
Minor edits, but otherwise this is good for sign off!

**Comment by reknai** [comment]
@ShannonLeavitt  Any update on the merging of this PR? Thx!

**Comment by asudbring** [comment]
@reknai 
Please address the changes and feedback and then we can sign off for merge.
Thanks!

**Comment by reknai** [comment]
@asudbring : the suggestions have been incorporated. Please check. Thank you!

**Comment by asudbring** [comment]
> @asudbring : the suggestions have been incorporated. Please check. Thank you!

@reknai
I'm not seeing the changes or a new build happening.  You can commit the suggested changes in the comments and it will automatically commit them and rebuild the PR.  The last commit I'm seeing on this PR was 10/8.  If you made the changes locally you have to push them up to Github for this PR.

**Comment by reknai** [comment]
@asudbring : I see it was not committed and I somehow missed it. Sorry about that. Now it should be in.

**Comment by asudbring** [comment]
#sign-off

**Pull request merged**